apps/explorer/src/app/routes/parties/id/components/party-id-error.tsx

```diff
@@ -14,7 +14,7 @@ type PartyIdErrorProps = {
 const PartyIdError = ({ id, error }: PartyIdErrorProps) => {
   const end = isValidPartyId(id)
     ? t('No accounts or transactions found for: ')
-    : 'Invalid party id: ';
+    : 'Invalid pubkey: ';
```

Comment:
Suggestion (non-blocking): for consistency's sake, this one should be 'public key' too

Reply:
Agreed!